openqa-maintenance.py

```diff
@@ -1029,6 +1026,10 @@ def setup_checker(self):
         return bot
 
 if __name__ == "__main__":
+    requests_log = logging.getLogger("requests.packages.urllib3")
+    requests_log.setLevel(logging.WARNING)
+    requests_log.propagate = False
+
```

Comment:
Hmm, not sure this belongs here. Maybe we need to leverage
https://docs.python.org/2/library/logging.config.html#logging-config-api